plugins/org.jkiss.dbeaver.ext.cubrid/src/org/jkiss/dbeaver/ext/cubrid/model/CubridSQLDialect.java

```diff
@@ -0,0 +1,11 @@
+package org.jkiss.dbeaver.ext.cubrid.model;
+
+import org.jkiss.dbeaver.ext.generic.model.GenericSQLDialect;
+
+public class CubridSQLDialect extends GenericSQLDialect {
+	public static final String CUBRID_DIALECT_ID = "cubrid";
+
+	public CubridSQLDialect() {

```

Comment:
Does Cubrid have specific keywords words? Or functions words. They can be added to this class.

Reply:
Currently, I don't have specific keywords or functions yet.

Reply:
Ok